config/settings.yml

```diff
@@ -129,83 +129,89 @@
-          :heat: notifications.*
-        :duration: 10.seconds
-        :capacity: 50
-        :amqp_port: 5672
-        :amqp_heartbeat: 30
-        :amqp_recovery_attempts: 4
-        :amqp_vhost: '/'
-        :ceilometer:
-          :event_types_regex: '\A(?!firewall|floatingip|gateway|net|port|router|subnet|security_group|vpn|volume|snapshot|backup)'
-        :stf:
-          :topic_name: anycast/ceilometer/event.sample
-          :event_types_regex: '\A(aggregate|compute\.instance|identity\.project|image|keypair|orchestration\.stack|servergroup|snapshot)'
-      :event_catcher_openstack_cinder:
-        :poll: 15.seconds
-        :topics:
-          :nova: notifications.*
-          :cinder: notifications.*
-          :glance: notifications.*
-          :heat: notifications.*
-        :duration: 10.seconds
-        :capacity: 50
-        :amqp_port: 5672
-        :amqp_heartbeat: 30
-        :amqp_recovery_attempts: 4
-        :amqp_vhost: '/'
-        :ceilometer:
-          :event_types_regex: '\A(volume|snapshot|backup)'
-        :stf:
-          :topic_name: anycast/ceilometer/event.sample
-          :event_types_regex: '\A(volume|snapshot|backup)'
-      :event_catcher_openstack_infra:
-        :poll: 15.seconds
-        :topics:
-          :nova: notifications.*
-          :cinder: notifications.*
-          :glance: notifications.*
-          :heat: notifications.*
-          :ironic: notifications.*
-        :duration: 10.seconds
-        :capacity: 50
-        :amqp_port: 5672
-        :amqp_heartbeat: 30
-        :amqp_recovery_attempts: 4
-        :amqp_vhost: '/'
-        :ceilometer:
-          :event_types_regex: '\A(?!firewall|floatingip|gateway|net|port|router|subnet|security_group|vpn)'
-      :event_catcher_openstack_network:
-        :poll: 15.seconds
-        :topics:
-          :neutron: "notifications.*"
-        :duration: 10.seconds
-        :capacity: 50
-        :amqp_port: 5672
-        :amqp_heartbeat: 30
-        :amqp_recovery_attempts: 4
-        :amqp_vhost: '/'
-        :ceilometer:
-          :event_types_regex: '\A(firewall|floatingip|gateway|net|port|router|subnet|security_group|security_group_rule|vpn)'
-        :stf:
-          :topic_name: anycast/ceilometer/event.sample
-          :event_types_regex: '\A(network|port|router|subnet|security_group|security_group_rule)'
-      :event_catcher_openstack_service: "auto"
-    :queue_worker_base:
-      :ems_metrics_collector_worker:
-        :ems_metrics_collector_worker_openstack: {}
-        :ems_metrics_collector_worker_openstack_infra: {}
-        :ems_metrics_openstack_default_service: "auto"
-        :ems_metrics_gnocchi_granularity: 300
-      :ems_refresh_worker:
-        :ems_refresh_worker_openstack: {}
-        :ems_refresh_worker_openstack_infra: {}
-        :ems_refresh_worker_openstack_network: {}
+  :ems_metrics_collector_worker:
+    :ems_metrics_gnocchi_granularity: 300
+    :ems_metrics_openstack_default_service: "auto"
```

Comment:
@agrare As discussed we should open a separate issue to not "infect" the base setting, but instead use the `ems.ems_openstack` section for these.

Reply:
wondering if these should become settings per provider.  in general, wondering whether global settings for provider workers makes sense.

Reply:
Ideally, yes, these should be per-provider-instance settings, though unfortunately I don't think we don't have a mechanism for that at the moment (without making it part of the "create provider" form)